documentation/docs/examples/clipping.mdx

```diff
@@ -70,42 +71,77 @@ setInterval(() => {
 :::
 
-### CAD geometry filter
+### Interactable slicing planes
+
+Often it's required to have interactive slicing planes that the user can modify. The following example 
+show how [ThreeJS DragControls](https://threejs.org/docs/#examples/en/controls/DragControls) can be used 
+to manipulate the slicing planes.
+
+After running this snippet the slicing plane can be manipulated by dragging the white manipulator.
 
 ```js runnable
 // import { DragControls } from 'three/examples/jsm/controls/DragControls';
 
 const bounds = model.getModelBoundingBox();
 
 // Create draggable cone at center of bounds
-const center = bounds.getCenter(new THREE.Vector3());
-var geometry = new THREE.BoxBufferGeometry(10, 10, 10);
-var mesh = new THREE.Mesh(geometry, new THREE.MeshBasicMaterial({ color: 'green' }));
-mesh.position.copy(center);
-viewer.addObject3D(mesh);
-
-// Prepare a clipping plane
+const initialDragPosition = new THREE.Vector3(bounds.min.x, 0.5*(bounds.max.y + bounds.min.y), bounds.min.z);
+const draggerGeometry = new THREE.SphereBufferGeometry(5);
+const draggerMaterial = new THREE.MeshBasicMaterial({ color: 'white' });
+const draggerMesh = new THREE.Mesh(draggerGeometry, draggerMaterial);
+draggerMesh.position.copy(initialDragPosition);
+viewer.addObject3D(draggerMesh);
+
+// Prepare a slicing plane
 const orientation = new THREE.Vector3(0, -1, 0);
-const point = new THREE.Vector3(0, center.y, 0);
 const plane = new THREE.Plane();
-plane.setFromNormalAndCoplanarPoint(orientation, point);
+plane.setFromNormalAndCoplanarPoint(orientation, initialDragPosition);
 viewer.setSlicingPlanes([plane]);
 
-const controls = new DragControls([mesh], viewer.getCamera(), viewer.domElement);
+// Handle drag events
+const controls = new DragControls([draggerMesh], viewer.getCamera(), viewer.domElement);
 controls.addEventListener('drag', () => {
   // Fix XZ-poistion
-  mesh.position.set(center.x, mesh.position.y, center.z);
+  draggerMesh.position.set(initialDragPosition.x, draggerMesh.position.y, initialDragPosition.z);
   // Update slicing plane
-  plane.setFromNormalAndCoplanarPoint(new THREE.Vector3(0, -1, 0), mesh.position);
+  plane.setFromNormalAndCoplanarPoint(new THREE.Vector3(0, -1, 0), draggerMesh.position);
   viewer.forceRerender();
 });
-controls.addEventListener('dragstart', () => viewer.cameraControlsEnabled = false);
-controls.addEventListener('dragend', () => viewer.cameraControlsEnabled = true);
+controls.addEventListener('dragstart', () => {
+  draggerMaterial.color = new THREE.Color('gray');
+  viewer.cameraControlsEnabled = false;
+});
+controls.addEventListener('dragend', () => {
+  draggerMaterial.color = new THREE.Color('white');
+  viewer.cameraControlsEnabled = true;
+});
+```
+
+### CAD geometry filter
+
+A special type of slicing is to provide a "geometry filter" when loading the model. A geometry filter restricts the 
+area to load geometry for and is specified when the model is created.
+
+```js 
```

Comment:
is it intentionally non-runnable?

Reply:
Yes, because it must be applied on creation which we don't support now.